Content.Shared/RCD/Components/RCDDeconstructibleComponent.cs

```diff
@@ -0,0 +1,33 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.RCD.Components;
+
+[RegisterComponent, NetworkedComponent]
+[Access([])]
```

Comment:
what

Reply:
Whoops. It should be [Access(typeof(RCDSystem))]